apis/storagecontrol/v1alpha1/managedfolder_identity.go

```diff
@@ -52,6 +52,14 @@ func (p *ManagedFolderParent) String() string {
 	return "projects/" + p.ProjectID + "/buckets/" + p.BucketName
 }
 
+// Builds a new ManagedFolderIdentity from parent and resourceID
+func GetManagedFolderIdentity(parent *ManagedFolderParent, id string) *ManagedFolderIdentity {
```

Comment:
nit: We want to get rid of this kind of `ManagedFolderIdentity` builder so that we can guarantee the Identity is only created in `AdaptorForObject`.

Reply:
Changed to use the URL to create an unstructured object and then call AdapterForObject